packages/components/src/styles/components/application-state.scss

```diff
@@ -7,43 +7,85 @@
 // APPLICATION-STATE COMPONENT
 //
 
-$hds-application-state-padding: 12px 0;
+$hds-application-state-content-max-width: 480px;
 
 .hds-application-state {
-  width: 19.5rem;
-  max-width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: start;
+  width: fit-content;
```

Comment:
question, mainly for @majedelass: with this rule the `ApplicationState` doesn't have anymore a predefined width (before it was fixed). It means that in some edge cases, the component may visually "collapse" on itself. Do you reckon it would make sense to define a min-width (in code only), let's say `280px`, to prevent this? (we could also do something more sophisticated, like `min-width: min(320px, 100%)` so it's also responsive.

Reply:
Is this conventionally what is done with other components this approach or is this something new we're doing?

From Figma we decided against doing a min width because we didn't want to limit its usage to only pages. This could be added in cards, potentially. Are we _certain_ that if we define a min-width it wont disrupt placement of this component?

Reply:
> This could be added in cards

Yeah, in that case it's OK no min width, it's an edge case anyway.

@zamoore I am going to add a use case within a card to the showcase, so we have it covered.

Reply:
I think there's a sensible min-width, even in cards. 280 is probably reasonable imo.